src/main/java/ca/bc/gov/chefs/etl/forms/pcd/hrRecords/json/ClinicRecordsDetails.java

```diff
@@ -0,0 +1,177 @@
+package ca.bc.gov.chefs.etl.forms.pcd.hrRecords.json;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ClinicRecordsDetails {
```

Comment:
It seems a little odd to have the double plural. Maybe just ClinicRecordsDetail?

Reply:
The field in CHEFS is called clinicRecordDetails so I changed it to ClinicRecordDetails for consistency.